pkg/config/config.go

```diff
@@ -102,3 +111,43 @@ func (c *Config) getVfInfo(vfPci string) (string, int, error) {
 
 	return pf, vfID, nil
 }
+
+// handleBridgeConfig checks PF bridge configuration and set PFBridge options for PluginConfig.
+// the function checks that PF is attached to one of the bridges which explicitly set
+// in the configuration or to the default bridge, returns error if PF is not a part of a bridge
+func (c *Config) handleBridgeConfig(conf *localtypes.PluginConf) error {
+	if conf.Bridge == "" {
+		conf.Bridge = DefaultBridge
+	}
+	allowedBridgeNames := strings.Split(conf.Bridge, ",")
+	for i := range allowedBridgeNames {
+		allowedBridgeNames[i] = strings.TrimSpace(allowedBridgeNames[i])
+	}
+
+	pfLink, err := c.netlink.LinkByName(conf.PFName)
+	if pfLink == nil || err != nil {
+		return fmt.Errorf("failed to get link info for PF %s", conf.PFName)
+	}
+
+	pfBridgeLink, err := utils.GetBridgeByLink(c.netlink, pfLink)
```

Comment:
you just return a bridge based on uplink so why don't you call it bridgeLink?

Reply:
I renamed the func to "GetParentBridgeForLink". I think this name describes intent of this function quite nice. WDYT?